### ISSUE TITLE
Software Steering Council representative from JupyterHub

### ISSUE THREAD

**Issue by consideRatio** [jupyterhub/team-compass#544]
@afshin introduced us to the need for a Software Steering Council representative from JupyterHub. I copy pasted parts below from the Jul 19th JupyterHub team meeting notes:

> Current Jupyter steering council will be disbanded and replaced by new bodies. Each subproject should have a representative to vote on JEPs. Need a JupyterHub Software Steering Council representative.
>
> https://jupyter.org/governance/software_steering_council.html
>
> Also related to JupyterHub governance: https://github.com/jupyterhub/team-compass/issues/497

I opened this issue to represent our need to elect such representative in one way or another.

**Comment by betatim** [comment]
How should we go about this?

**Comment by choldgraf** [comment]
I think that barring no action, we should assume that @minrk is a this person as the team lead.

If @minrk does not want to serve in this role, or if somebody else really does want this role, I think we should discuss this in a team meeting.

**Comment by minrk** [comment]
Arg, I wrote a comment here days ago but apparently closed the tab before posting. Sorry!

I think the process is:

1. refresh our team memberships (#497):
  1. contact existing members and ask if they'd like to remain or change status
2. vote to appoint a representative (should this be the council or all team members? If it's not clear from our governance, let's make it clear!)

I am happy to serve as _interim_ representative while we do this, since I effectively serve that role already in the prior governance structure, if it helps. I'm also happy if we still want me to do it after resolving a vote, or if someone else wants to have the role that's fine, too.

**Comment by fperez** [comment]
We should clarify - the SSC representative is meant to _represent_ the voice of the subproject's team in the SSC, and therefore to support communication and coordination with all other subprojects in areas that can impact others. But there is no need, nor the assumption, that the SSC rep and any measure of "formal subproject leadership" are embodied in the same person.

Obviously it's important that the representative is active and knowledgeable in the project, because they need to communicate about the impact of decisions from other parts of Jupyter in their project, and vice-versa. But when acting in the SSC, they are meant to be carrying the voice of the entire subproject, and they are expected to consult back with their project when there is doubt, rather than act alone.

We also want to emphasize the value of allowing active members who aren't necessarily the "recognized lead" of a subproject to take this role, in that it's a way to grow new voices of leadership across Jupyter. We've discussed a lot how we need more paths to leadership in Jupyter, and we think this can be one.

In summary, it's wonderful if @minrk does this, and he's obviously perfectly suited for the role, we're not trying to criticize that. But we want you to think with a slightly bigger picture in mind, in ways that can open the conversation (now or for a later time, that's entirely your choice).

We'll be adding clarifying language to this effect to our docs shortly, btw.
